### ISSUE TITLE
Trailing unused function parameters

### ISSUE THREAD

**Issue by littledan** [devsnek/proposal-unused-function-parameters#3]
For a case like `function f(x,){}`, should the length be 1 or 2? This is already valid syntax (1), but it looks a bit like a trailing unused parameter (2).

**Comment by devsnek** [comment]
i imagine a trailing unused parameter would be `function f(x,,) {}`, same as how array elisions work

**Comment by littledan** [comment]
Yes, that seems like the logical solution, but I think this gets a bit confusing in the case of Array elision, and I'm not sure if it's great to extend this dual meaning of commas to a new context.
